src/plugins/explore/public/application/pages/traces/trace_details/trace_view.tsx

```diff
@@ -0,0 +1,305 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import React, { useEffect, useState, useMemo, useRef, useCallback } from 'react';
+import { i18n } from '@osd/i18n';
+import { EuiSpacer, EuiPanel, EuiLoadingSpinner, EuiResizableContainer } from '@elastic/eui';
+import './trace_view.scss';
+import { TraceTopNavMenu } from './public/top_nav_buttons';
+import { useOpenSearchDashboards } from '../../../../../../opensearch_dashboards_react/public';
+import { TracePPLService } from './server/ppl_request_trace';
+import { MountPoint } from '../../../../../../../core/public';
+import { transformPPLDataToTraceHits } from './public/traces/ppl_to_trace_hits';
+import { DataExplorerServices } from '../../../../../../data_explorer/public';
+import { generateColorMap } from './public/traces/generate_color_map';
+import { SpanDetailPanel } from './public/traces/span_detail_panel';
+import { ServiceMap } from './public/services/service_map';
+import { NoMatchMessage } from './public/utils/helper_functions';
+import { createTraceAppState } from './state/trace_app_state';
+import { SpanDetailSidebar } from './public/traces/span_detail_sidebar';
+
+export interface TraceDetailsProps {
+  setMenuMountPoint?: (mount: MountPoint | undefined) => void;
+}
+
+export const TraceDetails: React.FC<TraceDetailsProps> = ({ setMenuMountPoint }) => {
+  const {
+    services: { chrome, data, osdUrlStateStorage },
+  } = useOpenSearchDashboards<DataExplorerServices>();
+
+  // Initialize URL state management
+  const { stateContainer, stopStateSync } = useMemo(() => {
+    return createTraceAppState({
+      stateDefaults: {
+        traceId: '',
+        dataSourceId: '',
+        indexPattern: 'otel-v1-apm-span-*',
+        spanId: undefined,
+      },
+      osdUrlStateStorage: osdUrlStateStorage!,
+    });
+  }, [osdUrlStateStorage]);
+
+  // Get current state values and subscribe to changes
+  const [appState, setAppState] = useState(() => stateContainer.get());
```

Comment:
This component uses local state + URL state container instead of the Redux pattern used elsewhere in explore. Seems have limited integration with existing explore services (timefilter, query string manager, etc.)